Frontend/src/app/views/dashboard/report/report.component.scss

```diff
@@ -1,19 +1,11 @@
-.container {
-    padding-left: 10%;
-    padding-right: 10%;
-    padding-top: 30px;
-}
+@import "../../../../app/styles/display.scss";
 
-@media only screen and (max-width: 830px) {
-    .container {
-        padding-left: 5px;
-        padding-right: 5px;
-    }
+.container {
+    padding: 20px;
 }
 
 @media only screen and (max-width: 1000px) {
     .container {
-        padding-left: 5%;
-        padding-right: 5%;
+        padding: 40px
     }
```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives

Reply:
Padding removed